### DISCUSSION TITLE
Blank page when install canvas ui , when i navigate though /canvas-ui even if i have publish a post

### DISCUSSION THREAD

**Discussion by premy-punchu** [austintoddj/canvas#1397] [Troubleshooting]
when i navigate though /canvas-ui even if i have publish a post the link is http://127.0.0.1:8000/canvas-ui

**Comment by austintoddj** [comment]
@premy-punchu Are there console errors you can share?

**Reply by premy-punchu** [reply]
I have already troubleshoot it when running npm install and run dev I forget to finish with canvas.ui.dev

It's now good to good
